src/components/CountryDetail.vue

```diff
@@ -1,7 +1,17 @@
 <template>
-    <div v-if="data">
-      <p> In {{year}}, {{countryName}} had ... Amenia and ... protein data</p>
+    <div v-if="data">   
+    <b-jumbotron >
```

Comment:
Du kannst den jumbotron außen machen und das `<div v-if="data">` drinnen schachteln, dann sparst du etwas Code...

Reply:
hä? das habe doch geändert, du siehst alte version von dem? solche lange satz habe ich gelöscht und in jumbotron gepackt

Reply:
Nee ist die richtige Version (das rote ist ja was du gelöscht hast), ich meine nur, dass deine Struktur ineffizient ist. Du könntest machen:

```
<b-jumbotron>
  <div v-if="data>
  ...
  </div>
  <div v-else>
  ...
  </div>
</b-jumbotron>
```
Dann hast du nicht zwei Jumbotrons, das ist ein klein wenig effizienter